### DISCUSSION TITLE
Ping Times

### DISCUSSION THREAD

**Discussion by luis9430** [directus/directus#2780] [Archive]
Hi, I am using directus with vue js, I have directus on localhost (xampp) but my latency is usually 180-200 ms, when I go to page  it takes about 15 seconds to receive the information, my question is.
What does latency depend on?

**Comment by benhaynes** [comment]
Many factors! Most notably, the current version 8 uses a loop query to fetch relational data... which is absurdly slow for deeply nested data. The upcoming v9 avoids this and is much smarter about how it fetches data in general. Our speed benchmarks show an enormous improvement in v9 (as well as more robust caching)... so that will solve this issue.
